docker-entrypoint.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+# If command starts with an option, prepend postfwd2
+if [ "${1:0:1}" = '-' ]; then
+  set -- postfwd2 "$@"
+fi
+
+# Check if help, version or manual arguments were used
+want_help=
+for arg; do
+  case "$arg" in
+    --help|-h|--manual|-m|--version|-V)
+      want_help=1
+      break
+      ;;
+  esac
+done
+
+
+if [ "$1" = "postfwd1"  ] || [ "$1" = "postfwd2" ] && [ -z "$want_help" ]; then
+  if [ ! -f /etc/postfwd/anti-spam.conf ]; then
+    echo >&2 'ERROR: Anti-spam plugin configuration file not found. You probably forgot to mount it. Please check README.md for instructions.'
```

Comment:
hmm, i'd create simple usage message instead of referencing README.md. Anyway there is not a link for the readme.

Reply:
I will figure out something more usefull.